src/cljs/villagebookUI/components/create_category.cljs

```diff
@@ -0,0 +1,59 @@
+(ns villagebookUI.components.create-category
+  (:require [reagent.core :as r]
+            [villagebookUI.helpers :as helpers]
+            [villagebookUI.components.utils :as utils]
+            [villagebookUI.fetchers :as fetchers]
+            [villagebookUI.api.category :as api]
+            [villagebookUI.store.organisations :as org-store]))
+
+(defn submit-create-category
+  [name fields]
+  (let [org-id (:id (org-store/get-selected))]
+    (api/create {:name   name
+                 :fields (map #(second %) fields)}
+                org-id
+                #(do
+                   (fetchers/fetch-categories! org-id last)
+                   (helpers/show-alert-bottom! :success (str "Category " name " created")))
+                #(helpers/show-alert-bottom! :error (:response %)))))
+
```

Comment:
[murtaza] Consider adding all of these parameters to a single map, and destructure it in the function.